src/explore-education-statistics-common/src/modules/find-statistics/components/DataBlockTabs.tsx

```diff
@@ -143,7 +228,7 @@ const DataBlockTabs = ({
             }
             title="Table"
           >
-            {!!error && errorMessage}
+            {!!tableDataError && errorMessage}
```

Comment:
Don't need this if we display the `errorMessage` if `isTableDataError` is true before the tabs render (see earlier comment).